src/main/scala/de/bripkens/ha/App.scala

```diff
@@ -7,28 +7,30 @@ import com.fasterxml.jackson.databind.JsonMappingException
 
 import scala.util.{Failure, Success}
 
-object App extends scala.App {
-
-  if (args.length != 1) {
-    reportCriticalInitialisationError(
-      "Please specify exactly one parameter: The path to the config file."
-    )
-  }
-
-  val configPath = args(0)
-  Console.out.println(s"Starting with config file $configPath")
-
-  Configuration.load(Paths.get(configPath)) match {
-    case Success(configuration) => startActorSystem(configuration)
-    case Failure(e: NoSuchFileException) => reportCriticalInitialisationError(
-      s"Config file $configPath does not exist."
-    )
-    case Failure(e: JsonMappingException) => reportCriticalInitialisationError(
-      s"Config file $configPath could not be parsed. Error: ${e.getMessage}"
-    )
+object App {
+
+  def main(args: Array[String]) {
+    if (args.length != 1) {
+      reportCriticalInitialisationError(
+        "Please specify exactly one parameter: The path to the config file."
+      )
+    }
+
+    val configPath = args(0)
```

Comment:
Would be simpler, but would change semantics. Currently we're checking for exactly one argument. With `headOption` it would check for 1 or more. Thoughs @bripkens?